labelbox/schema/task.py

```diff
@@ -68,12 +74,44 @@ def wait_till_done(self, timeout_seconds=300) -> None:
             time.sleep(sleep_time_seconds)
             self.refresh()
 
-    def errors(self):
+    @property
+    def errors(self) -> Optional[Dict[str, Any]]:
         """ Downloads the result file from Task
         """
-        if self.status == "FAILED" and self.result:
-            response = requests.get(self.result)
-            response.raise_for_status()
-            data = response.json()
-            return data.get('error')
+        if self.status == "FAILED":
+            result = self._fetch_remote_json()
+            return result['error']
```

Comment:
result.get('error')?

Reply:
I was thinking that this should never be None since we are specifically checking that the action is Json import.